runusb

```diff
@@ -159,29 +168,28 @@ class USBHandler(metaclass=ABCMeta):
 
 class RobotUSBHandler(USBHandler):
     def __init__(self, mountpoint_path: str) -> None:
+        self._setup_logging(mountpoint_path)
         LED_CONTROLLER.yellow()
         env = dict(os.environ)
         env["SBOT_METADATA_PATH"] = MOUNTPOINT_DIR
         self.process = subprocess.Popen(
-            [
-                "script",
-                "--quiet",
-                "--flush",
-                "--return",
-                "--command",
-                f"python3 {ROBOT_FILE}",
-                "log.txt",
-
-            ],
+            [sys.executable, '-u', ROBOT_FILE],
             stdin=subprocess.DEVNULL,
+            stderr=subprocess.STDOUT,
+            stdout=subprocess.PIPE,
+            bufsize=1,  # line buffered
```

Comment:
Question: Why are we setting a buffer here, but running Python unbuffered?

Reply:
This means only one line is buffered total across all the layers. I had issues while testing around readline and being fully unbuffered. 
Since the logger flushes to disk after each message we should at most only loose the final line.

Reply:
Weird. If this works, then one to dig further into another time.